apps/vor/pkg/postgres/schoolStore.go

```diff
@@ -62,7 +65,52 @@ func (s SchoolStore) GetStudents(schoolId string) ([]Student, error) {
 	}
 	return students, nil
 }
+func (s SchoolStore) GetClassAttendance(classId string, session string) ([]Attendance, error) {
+	var attendance []Attendance
+	if session == "" {
+		session = "1970-01-01"
+	}
+	if err := s.Model(&attendance).
+		Where("class_id=?", classId).
+		Where("date::date>=?", session).
+		Relation("Student").
+		Relation("Class.Students").
+		Select(); err != nil {
+		return nil, err
+	}
+	return attendance, nil
+}
+func (s SchoolStore) GetClassSession(classId string) ([]ClassSession, error) {
```

Comment:
CMIIW, this function would return all session on attendance table and all of the next sessions for the following week?

Reply:
This is more of what i have in mind:
![excalidraw-2020314193352](https://user-images.githubusercontent.com/8491552/80872248-f8041e00-8cda-11ea-92c7-295600943c6f.png)

Reply:
Okay then this will be the same logic with getClass session. The difference is that this endpoint return attendance details, while getClassSession only returns the date

Reply:
Yes, that is correct.

Reply:
For this endpoint, do we enable to fetch only data for 1 session? Or we just return everything?

Reply:
Are we talking about the getAttendance endpoint?